packages/reports/app/styles/navi-reports/components/schedule-modal.scss

```diff
@@ -0,0 +1,55 @@
+/**
+* Copyright 2021, Yahoo Holdings Inc.
+* Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+*/
+.modal.is-active {
+  visibility: visible !important;
+}
```

Comment:
The way this is written it would impact all modals, not just schedule. I think is already fixed in the directory.

Reply:
Without this the tests in reports fail and when you move your mouse off the modal it disappears.